### PR TITLE
Move ballerina-internal unit tests

### PR DESCRIPTION
**Pull request by riyafa**

### CODE REVIEW

File: stdlib/internal/src/test/resources/logging.properties [hunk 1]
```diff
@@ -0,0 +1,37 @@
+#
+# Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (Bhashinee) [inline stdlib/internal/src/test/resources/logging.properties]: Shouldn't this be 2019?

File: stdlib/internal/src/test/resources/datafiles/compression/my.app/test/hello.bal [hunk 2]
```diff
@@ -0,0 +1,3 @@
+public function greet() returns (string){
```
Comment (wggihan) [inline stdlib/internal/src/test/resources/datafiles/compression/my.app/test/hello.bal]:
```suggestion
public function greet() returns string { 
```

File: stdlib/internal/src/test/resources/datafiles/compression/my.app/test/test_1/hello.bal [hunk 3]
```diff
@@ -0,0 +1,3 @@
+public function greet() returns (string){
```
Comment (wggihan) [inline stdlib/internal/src/test/resources/datafiles/compression/my.app/test/test_1/hello.bal]:
```suggestion
public function greet() returns string { 
```

File: stdlib/internal/src/test/resources/testng.xml [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+under the License.
+-->
+
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
+
+<suite name="ballerina-test-suite">
+
+    <!-- Ballerina language Test Cases. -->
+    <test name="ballerina-lang-test-suite" preserve-order="true" parallel="false">
+        <groups>
```
Comment (wggihan) [inline stdlib/internal/src/test/resources/testng.xml]: Are there any broken tests in this module? If not, shall we remove this broken group?

**Pull request merged**